dali/operators/generic/cast.cc

```diff
@@ -26,6 +26,7 @@ void Cast<CPUBackend>::RunImpl(SampleWorkspace &ws) {
   DALIDataType itype = input.type().id();
 
   TYPE_SWITCH(output_type_, type2id, OType, CAST_ALLOWED_TYPES, (
+    output.SetLayout(input.GetLayout());
```

Comment:
Please rebase. I think is has been merged already.

Reply:
done